pkg/sypgp/sypgp.go

```diff
@@ -494,6 +516,27 @@ func (keyring *Handle) RemovePubKey(toDelete string) error {
 	return keyring.storePubKeyring(newKeyList)
 }
 
+// RemovePrivKey will delete a private key matching toDelete
+func (keyring *Handle) RemovePrivKey(toDelete string) error {
+	elist, err := loadKeyring(keyring.SecretPath())
+	switch {
+	case os.IsNotExist(err):
+		return nil
+
+	case err != nil:
+		return fmt.Errorf("unable to list local secret keyring: %v", err)
+	}
+
+	newKeyList := removeKey(elist, toDelete)
+	if newKeyList == nil {
+		return fmt.Errorf("no key matching given fingerprint found")
+	}
+
+	sylog.Verbosef("updating local secret keyring: %v", keyring.SecretPath())
```

Comment:
Just a missing uppercase here

Reply:
@cclerget thanks for the comment. Resolved the comment in this commit
[`ab65928` (#555)](https://github.com/apptainer/apptainer/pull/555/commits/ab65928bb0818a107ab8f68af1245bca498ef2ef)